docs/stylesheets/extra.css

```diff
@@ -145,7 +145,7 @@ code::first-of-type and code::last-of-type should get left and right padding res
 
 figure {
   border: thin solid var(--md-typeset-a-color);
-  background-color: #eee;
```

Comment:
I happened to look at this figure this evening, while my browser was in dark mode. It looks like this:

![image](https://github.com/HGVSnomenclature/hgvs-nomenclature/assets/811065/8399fd91-17f8-4715-8193-4c4d732216bd)

I propose keeping the background styling for figures white–or at least doing so for this figure.